recipes/otree/recipe.yaml

```diff
@@ -0,0 +1,50 @@
+context:
+  version: "0.3.1"
+
+package:
+  name: otree
+  version: ${{ version }}
+
+source:
+  url: https://github.com/fioncat/otree/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: f71064cac1c7fc1047a119bdd2b630eac67008546d13b51b1ace1ab9b0314e02
+
+build:
+  number: 0
+  script:
+    - if: unix
+      then:
+        - cargo install --no-track --locked --root ${PREFIX} --path .
+        - cargo-bundle-licenses --format yaml --output ${SRC_DIR}/THIRDPARTY.yml
+    - if: win
+      then:
+        - cargo install --no-track --locked --root %PREFIX% --path .
+        - if errorlevel 1 exit 1
+        - cargo-bundle-licenses --format yaml --output %SRC_DIR%/THIRDPARTY.yml
+        - if errorlevel 1 exit 1
+
+requirements:
+  build:
+    - ${{ compiler('rust') }}
+    - ${{ stdlib('c') }}
+    - ${{ compiler('c') }}
+    - cargo-bundle-licenses
+
+tests:
+  - script:
+      - otree --help
+
+about:
+  homepage: https://github.com/fioncat/otree
+  repository: https://github.com/fioncat/otree
+  documentation: https://github.com/fioncat/otree
+  summary: A command line tool to view objects (JSON/YAML/TOML) in TUI tree widget.
+  license: MIT
+  license_file:
+    - LICENSE
+    - THIRDPARTY.yml
+    - CONDA_BUILD_LICENSE.txt
```

Comment:
I haven't seen the `CONDA_BUILD_LICENSE.txt` included before. What's the purpose of this?

Reply:
My mistake. This is from a recipe I used as a template. I'll remove it.